### PR TITLE
Removed WLS logging export config from ELK stack fluentd test suite

### PR DESCRIPTION
**Pull request by hzhao-github**
Jenkins job:

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10472
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/10473

### CODE REVIEW

**Review by anpanigr — COMMENTED** [review]
There is a plan to remove WebLogic logging exporter tests from the integration test, as we will be no longer counting the project weblogic logging exporter. Let us confirm with Monica and remove WebLogic Logging Exporter tests.

**Comment by hzhao-github** [comment]
ok. if that's true, we should just modify existing wdt file for both test suites

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java [hunk 1]
```diff
@@ -472,4 +477,50 @@ private String execSearchQuery(String queryCriteria, String index) {
 
     return execResult.stdout();
   }
+
+  private static void modifyModelConfigfile() {
+    final String sourceConfigFile = MODEL_DIR + "/model.wlslogging.yaml";
+
+    assertDoesNotThrow(() -> copy(Paths.get(sourceConfigFile), Paths.get(WLS_LOGGING_MODEL_FILE)),
+        "copy model.wlslogging.yaml failed");
```
[the remark this answers is not in the document]
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java:485]: I am using FileUtils.copy, that handles StandardCopyOption  options :)

[the remark this answers is not in the document]
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java:485]: ok

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**